src/style.css

```
@@ -1,3 +1,99 @@
+@font-face {
+  font-family: Roboto;
+  src: local('Roboto'), local('Roboto-Regular'),
+    url(https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
+}
+
+:root {
+  --link-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+
+  display: flex;
+
+  justify-content: space-between;
+
+  align-items: center;
+}
+
+.header_section_logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.header_section_nav {
+  margin-right: 50px;
+}
```

Remove it and add padding for the header
```suggestion
}

```